### PR TITLE
Reference all available standard libraries

### PR DESCRIPTION
**Pull request by schellap**

### CODE REVIEW

File: src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ private void InitializeArgs(NativeCompileSettings config)
             argsList.Add($"\"{inputFilePath}\"");
             
             // System.Private.CoreLib Reference
-            var coreLibPath = Path.Combine(config.IlcSdkPath, "sdk", "System.Private.CoreLib.dll");
+            var coreLibPath = Path.Combine(config.IlcSdkPath, "sdk", "*.dll");
```
Comment (brthor) [inline src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs]: Is this expected to change often? I am wondering if an enumeration would not be more reliable?

Comment (schellap) [inline src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs]: These are like standard libraries and we may add more to the list, and if we refactored code and made more DLLs, we would have to keep updating the CLI.

**Comment by schellap** [comment]
See this corresponding PR https://github.com/dotnet/corert/pull/596

File: src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs (hunk 1 above)
Comment (brthor) [inline src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs]: I see, yeah if this is going to change often * makes sense. :+1:

**Comment by schellap** [comment]
@dotnet-bot test OSX Debug Build please

**Comment by schellap** [comment]
@dotnet-bot test OSX Release Build please

**Pull request merged**